### ISSUE TITLE
Viewer won't resize as window resizes

### ISSUE THREAD

**Issue by Desperationis** [Lux-AI-Challenge/LuxViewer2021#8]
If you start the viewer in full screen and upload a replay, you get this, as expected: 
![full](https://user-images.githubusercontent.com/42013493/124362445-b118fd80-dbe9-11eb-9dcb-f4257ab357c8.png)

However, if you start the viewer in a small window, upload a replay, then resize the window, the game area is cut off at the width and height of the small window: 
![cut](https://user-images.githubusercontent.com/42013493/124362459-d60d7080-dbe9-11eb-8078-26a01927e0e4.png)

**Comment by StoneT2000** [comment]
Yeah this is a weird bug that's been in our internal backlog for a while. Not sure when we will get to this ( just don't resize 😃 )
